### PR TITLE
Feature/add data initializer

### PR DESCRIPTION
**Pull request by zamiramanasova**
added data initializer in DB and  swagger in course controller

### CODE REVIEW

File: src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    @Override
+    @Transactional
+    public void onApplicationEvent(ContextRefreshedEvent event) {
+        Role adminRole = new Role("ROLE_ADMIN");
+        User user1 = new User();
+        user1.setFirstName("Aaaa");
+        user1.setLastName("Aaaa");
+        user1.setPassword(passwordEncoder.encode("Aaaa"));
+        user1.setEmail("aaaa@test.com");
```
Comment (ChyngyzIshen) [inline src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java:42]: admin@gmail.com https://peaksoftschool.atlassian.net/jira/software/projects/LM/pages

File: src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    }
+
+    @Override
+    @Transactional
+    public void onApplicationEvent(ContextRefreshedEvent event) {
+        Role adminRole = new Role("ROLE_ADMIN");
+        User user1 = new User();
+        user1.setFirstName("Aaaa");
+        user1.setLastName("Aaaa");
+        user1.setPassword(passwordEncoder.encode("Aaaa"));
```
Comment (ChyngyzIshen) [inline src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java:41]: admin

File: src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        user1.setRoles(Arrays.asList(adminRole));
+        user1.setCreated(LocalDateTime.now());
+        userRepository.save(user1);
+
+        Role instructorRole = new Role("ROLE_INSTRUCTOR");
+        User user2 = new User();
+        user2.setFirstName("Bbbb");
+        user2.setLastName("Bbbb");
+        user2.setPassword(passwordEncoder.encode("bbbb"));
+        user2.setEmail("bbbb@test.com");
```
Comment (ChyngyzIshen) [inline src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java:52]: add instructor@gmail.com

File: src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+        user1.setEmail("aaaa@test.com");
+        user1.setRoles(Arrays.asList(adminRole));
+        user1.setCreated(LocalDateTime.now());
+        userRepository.save(user1);
+
+        Role instructorRole = new Role("ROLE_INSTRUCTOR");
+        User user2 = new User();
+        user2.setFirstName("Bbbb");
+        user2.setLastName("Bbbb");
+        user2.setPassword(passwordEncoder.encode("bbbb"));
```
Comment (ChyngyzIshen) [inline src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java:51]: instructor

File: src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+        user2.setRoles(Arrays.asList(instructorRole));
+        user2.setCreated(LocalDateTime.now());
+        userRepository.save(user2);
+
+        Role studentRole = new Role("ROLE_STUDENT");
+        User user3 = new User();
+        user3.setFirstName("Cccc");
+        user3.setLastName("Cccc");
+        user3.setPassword(passwordEncoder.encode("cccc"));
+        user3.setEmail("cccc@test.com");
```
Comment (ChyngyzIshen) [inline src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java:62]: add student@gmail.com

File: src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+        user2.setEmail("bbbb@test.com");
+        user2.setRoles(Arrays.asList(instructorRole));
+        user2.setCreated(LocalDateTime.now());
+        userRepository.save(user2);
+
+        Role studentRole = new Role("ROLE_STUDENT");
+        User user3 = new User();
+        user3.setFirstName("Cccc");
+        user3.setLastName("Cccc");
+        user3.setPassword(passwordEncoder.encode("cccc"));
```
Comment (ChyngyzIshen) [inline src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java:61]: student

**Review by ChyngyzIshen — COMMENTED** [review]
comments

File: src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java (hunk 1 above)
Comment (zamiramanasova) [reply src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java]: исправила

File: src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java (hunk 3 above)
Comment (zamiramanasova) [reply src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java]: changed

File: src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java (hunk 4 above)
Comment (zamiramanasova) [reply src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java]: changed

File: src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java [hunk 7]
```diff
@@ -36,30 +36,30 @@ public DemoData(UserRepository userRepository, RoleRepository roleRepository, Pa
...
         user1.setCreated(LocalDateTime.now());
         userRepository.save(user1);
 
         Role instructorRole = new Role("ROLE_INSTRUCTOR");
         User user2 = new User();
-        user2.setFirstName("Bbbb");
-        user2.setLastName("Bbbb");
-        user2.setPassword(passwordEncoder.encode("bbbb"));
-        user2.setEmail("bbbb@test.com");
+        user2.setFirstName("instrucotor");
```
Comment (ChyngyzIshen) [inline src/main/java/kg/peaksoft/peaksoftlmsm1/api/DemoData.java:49]: should be instructor

**Review by ChyngyzIshen — COMMENTED** [review]
comment

**Pull request merged**